sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/MySQLSchemaReconciler.java

```diff
@@ -40,6 +42,13 @@
 
   public MySQLSchemaReconciler() {}
 
+  @Override
+  public Optional<Object> configurationFor(DependentResourceSpec<?, ?> dependent) {
+    return SchemaDependentResource.class.equals(dependent.getDependentResourceClass())
+        ? Optional.of(new ResourcePollerConfig(500, MySQLDbConfig.loadFromEnvironmentVars()))
```

Comment:
This could be just the default in the dependent resource `configureFor` method. Don't think we need this at all.